### PR TITLE
refactor(sbb-expansion-panel): improve expansion animation

### PR DESCRIPTION
**Pull request by dauriamarco**
Refactor `sbb-expansion-panel` animation.

### CODE REVIEW

**Review by jeripeierSBB — APPROVED** [review]
Love it! :)

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Nice work 😃

File: src/components/sbb-expansion-panel/sbb-expansion-panel.scss [hunk 1]
```diff
@@ -8,10 +8,10 @@
   --sbb-expansion-panel-animation-duration: var(--sbb-animation-duration-4x);
```
Comment (jeripeierSBB) [inline src/components/sbb-expansion-panel/sbb-expansion-panel.scss:8]: Please don't forget to remove [data-resize-disable-animation] in CSS

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]

**Comment by dauriamarco** [comment]
@jeripeierSBB I also checked that we had no problem with the content overflow as in the selection panel and we should be safe here. 👍

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**